ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -33,6 +34,12 @@ def symmetrize(x):
     return ivy.eigh(x, UPLO=UPLO)
 
 
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"0.4.14 and below": ("float16",)}, "jax")
+def qr(x, /, *, full_matrices="reduced"):
```

Comment:
here we need to adhere the frontend implementation so the parameter full_matrices is a bool argument so you need to handle if it is True then full_matrices = "reduced else it will be 'complete' have a lookat the function doc here (https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.linalg.qr.html)